CRM/Civicase/Helper/NewCaseWebform.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/**
+ * CRM_Civicase_Helper_NewCaseWebform class.
+ */
+class CRM_Civicase_Helper_NewCaseWebform {
+
+  /**
+   * Retrieve civicase webform url.
+   */
+  public static function appendUrlToOptions(&$options) {
```

Comment:
This function adds the webformURL and the webformClient to the options array. I think a better name will be 
`addWebformDataToOptions` or something better, we should change the function description to reflect that also.